app/controllers/organizations/field_of_studies_controller.rb

```diff
@@ -0,0 +1,61 @@
+module Organizations
+  class FieldOfStudiesController < ApplicationController
+    include FlashPreparer
+
+    expose :field_of_study
+    expose :organization, id: -> { params[:organization_id] }
+    expose :field_of_studies, -> { organization.field_of_studies.order(updated_at: :desc) }
+
+    before_action :authorize_field_of_study, only: %i[create update destroy]
```

Comment:
I'm not sure, but I think we can call method like that here:
```suggestion
    before_action { authorize! field_of_study }, only: %i[create update destroy]
```

Reply:
Looks like i can't writ like this, but i can write

```
    before_action only: %i[create update destroy] do
      authorize! field_of_study
    end
```

I don't really like this implementation